hr_worked_days_from_timesheet/views/hr_payslip_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+     <record id="view_hr_payslip_form" model="ir.ui.view">
+        <field name="name">hr.payslip.form</field>
+        <field name="model">hr.payslip</field>
+        <field name="inherit_id" ref="payroll.hr_payslip_view_form" />
+        <field name="priority">400</field>
+        <field name="arch" type="xml">
+            <field name="worked_days_line_ids" position="before">
+                <group>
+                    <separator string="Worked Days" />
+                    <div class="oe_right oe_button_box">
+                        <button
+                            string="Import from Timesheets"
+                            name="import_worked_days"
+                            type="object"
+                            states="draft"
+                        />
+                    </div>
+                </group>
+            </field>
+            <xpath expr="//sheet/notebook/page[1]/separator[1]" position="replace" />
```

Comment:
Why is this needed?

Reply:
I'ts invisible now, the replace was just to hide the original separator

Reply:
ok!